.github/workflows/post_publish.yml

```
@@ -55,28 +55,30 @@ jobs:
   registry-check:
     runs-on: ubuntu-latest
     needs: [on-success-or-workflow-dispatch]
+    outputs:
+      latest-version: ${{ steps.registry_latest_ver.outputs.current }}
     steps:
       - name: Registry Version Check
         id: registry_latest_ver
         shell: bash
         run: |
-          LATEST_VERSION=$(curl -s "https://registry.terraform.io/v2/providers/323/provider-versions/latest" | jq -r '.data.attributes.version')
-          if [[ " ${{ needs.on-success-or-workflow-dispatch.outputs.release-tag }}" != ${LATEST_VERSION} ]]; then
-            case "$OSTYPE" in
-              darwin*) sleep 1h ;;
-              linux*) sleep 1h ;;
-              msys*) Start-Sleep -s 3600
-              *) echo "unknown: $OSTYPE" ;;
-            esac
-          else
-            echo "Registry and Github Version matches"
-          fi
+          for i in 1 2
+          do
+            LATEST_VERSION=$(curl -s "https://registry.terraform.io/v2/providers/323/provider-versions/latest" | jq -r '.data.attributes.version')
+            if [[ " ${{ needs.on-success-or-workflow-dispatch.outputs.release-tag }}" != ${LATEST_VERSION} ]]; then
+              sleep 1h
+            else
+              echo "Registry and Github Version matches"
+                echo "current=$LATEST_VERSION >> "$GITHUB_OUTPUT"
+            fi
+          done
```

We probably want this job to fail if `$LATEST_VERSION` and `needs.on-success-or-workflow-dispatch.outputs.release-tag` never end up matching, that way `os-version-init` won't run.

If we're introducing a failure here, do you think we should do a Slack notification, like you've added for the `os-version-init` job? I'm unsure of how chatty we want these to be.

```suggestion
            else
                echo "Registry and Github Version matches"
                echo "current=$LATEST_VERSION" >> "$GITHUB_OUTPUT"
                exit
            fi
        done

        echo "Registry does not contain ${{ needs.on-success-or-workflow-dispatch.outputs.release-tag }}"
        exit 1
```